artemis-distribution/pom.xml

```diff
@@ -183,6 +183,73 @@
            <artifactId>tomcat-servlet-api</artifactId>
        </dependency>
 
+      <!-- for artemis-quorum-ri -->
```

Comment:
wheres the ZK alternative option dependencies?

Reply:
eagle eyes :D adding them now, together with features

Reply:
I still dont spot them. Unresolving. I could be being blind

Reply:
see https://github.com/apache/activemq-artemis/pull/3555/files#r625567290 and https://github.com/apache/activemq-artemis/pull/3555/files#r625528887